app/src/lib/stores/app-store.ts

```diff
@@ -3593,7 +3593,7 @@ export class AppStore extends TypedBaseStore<IAppState> {
       eligibleBranches =
         defaultBranch != null &&
         eligibleForFastForward(defaultBranch, currentBranchName)
-          ? [defaultBranch]
+          ? [defaultBranch].concat(state.branchesState.recentBranches)
```

Comment:
Ah, we should probably de-dupe the default branch if it's also one of the recent branches. Will do... I think I saw a `merge` function somewhere being used for this 🤔

Reply:
Nevermind, turns out `recentBranches` is exclusive of `defaultBranch`. So we're good 👍